WebApp/autoreduce_webapp/templates/runs_list.html

```diff
@@ -1,4 +1,5 @@
 {% extends "base.html" %}
```

Comment:
It wont let me comment on the exact lines, but do we want inline javascript, and `var` -> `let` and missing camelCase and `new_url` is redundant instead `document.location.href = window.location.origin + window.location.pathname + '?page=' + arguments[0] + '&filter=' + filter_choice + '&pagination=' + pagination_choice + '&sort=' + sort_choice;`

Reply:
I agree, this should be moved to a separate js file. And I'll clean it up a bit while doing that.